dashbuilder/dashbuilder-client/dashbuilder-renderers/dashbuilder-renderer-c3/src/main/java/org/dashbuilder/renderer/c3/client/exports/ResourcesInjector.java

```diff
@@ -14,8 +12,6 @@ public static void ensureInjected() {
     }
 
     private static void injectAllResources() {
-        String c3css = NativeLibraryResources.INSTANCE.c3css().getText();
```

Comment:
I'm wondering if we can now also remove this c3css from [here](https://github.com/kiegroup/appformer/blob/b9f688c6a8fe64b719cf8f332412039d83e2afd3/dashbuilder/dashbuilder-client/dashbuilder-renderers/dashbuilder-renderer-c3/src/main/java/org/dashbuilder/renderer/c3/client/exports/NativeLibraryResources.java) since you removed the (only?) usage + the associate resource css file? I didn't find any other usage in kiegroup repos. This way nobody will have to wonder in the future "is anyone using this css file?"

Reply:
I completely removed it in my last commit, I kept in case patternfly is updated, but if that happens we can find this PR and bring back the c3 default CSS. Thanks for the suggestion @jhrcek